.github/ISSUE_TEMPLATE.md

```diff
@@ -5,8 +5,8 @@ This is a (multiple allowed):
 * [ ] feature-discussion (RFC)
 
 ## DESCRIPTIVE TITLE HERE
```

Comment:
Okay I will make those changes too.

What's the consensus about note for not using issues tracker for questions?